### PR TITLE
comment stakes needed to preboost/unpreboost

### PR DESCRIPTION
**Pull request by dkent600**
@jellegerbrandy , can you please review the accuracy of the comments I am proposing here?

### CODE REVIEW

File: src/proposal.ts [hunk 1]
```diff
@@ -511,25 +511,46 @@ export class Proposal implements IStateful<IProposalState> {
...
-      // note that the number can be negative!
+      /**
+       * The number of up-staking tokens (usually GEN) needed to qualify a queued proposal to move into the
+       * pre-boosted queue.
+       *
+       * Only computed for queued proposals.
+       *
+       * The equation is derived from: threshold = (stakesFor + upstakeNeededToPreBoost) / stakesAgainst
+       *
+       * >=  0 : any number of up-staking tokens greater than this will qualify to move the proposal
```
Comment (jellegerbrandy) [inline src/proposal.ts]: I do not really get what "this" denotes in these lines. What is `>=0` ?

I'd just remove them: I think the line above ("the number of up-stsakeing tokens needed to qualify...") explains it pretty well already.

File: src/proposal.ts [hunk 2]
```diff
@@ -511,25 +511,46 @@ export class Proposal implements IStateful<IProposalState> {
...
-      // note that the number can be negative!
+
+      /**
+       * The number of down-staking tokens (usually GEN) needed to qualify a pre-boosted proposal to move back
+       * to the Queued queue.
+       * Only computed for PreBoosted proposals.
+       *
+       * The equation is derived from: threshold = stakesFor / (stakesAgainst + downStakeNeededToQueue)
+       *
+       * >  0 : any number of down-staking tokens greater-than-or-equal to this will qualify to move the proposal
```
Comment (jellegerbrandy) [inline src/proposal.ts]: same comment as above :-)

File: src/proposal.ts (hunk 1 above)
Comment (dkent600) [reply src/proposal.ts]: I clarified the comments.  I think they provide more info than is readily available by thinking about the equations.

File: src/proposal.ts (hunk 2 above)
Comment (dkent600) [reply src/proposal.ts]: I clarified the comments.  I think they provide more info than is readily available by thinking about the equations.

**Pull request merged**